tools/container.bzl

```diff
@@ -1,11 +1,50 @@
-load("@io_bazel_rules_docker//container:container.bzl", "container_push")
+load("@aspect_bazel_lib//lib:transitions.bzl", "platform_transition_filegroup")
+load("@rules_oci//oci:defs.bzl", "oci_push", "oci_image", "oci_image_index")
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+
+def multiarch_go_image(name, binary):
+    """Create a container image with two variants of the given go_binary target.
+    
+    Args:
+        name: resulting oci_image_index target
+        binary: label of a go_binary target; it may be transitioned to another architecture
+    """
+    images = []
+    tar_target = "_{}.tar".format(name)
+    image_target = "_{}.image".format(name)
+
+    pkg_tar(
+        name = tar_target,
+        srcs = [binary],
+        include_runfiles = True,
```

Comment:
Shouldn't this specify /app as a package dir?

Reply:
yes, I pushed another commit right after this one, sorry you reviewed the intermediate state